docs/headless-api.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Rojo Plugin APIs
+sidebar_position: 7
+---
+
+The Rojo Studio plugin exposes a headless API into `_G`, which is very valuable to the Rojo ecosystem. This allows you to make companion plugins that interact with Rojo!
+
+### Accessing the API
+```Lua
+local Rojo = _G.Rojo
+```
+Plugins and command bar all share the same _G which allows us to put our interface in. Using the key `"Rojo"` is easy to remember and follows branding.
+
+### API Documentation
+
+**Functions:**
+
+```Lua
+Rojo:RequestAccess(apis: {string}): {[string]: boolean}
+```
+In order to use any of these headless APIs, you must first explicitly request them. Users will be prompted to allow/deny each API your plugin aims to use, and it will return a dictionary of the APIs to a boolean of whether it was granted or not. This must be the first thing your plugin calls when using this headless API feature. This keeps our users secure and protected from malicious plugins.
```

Comment:
Consider combining last 2 sentences: "In order to protect users from malicious plugins, RojoAccess must be the first function called by your plugin when using this headless API feature."